src/events.py

```
@@ -18,51 +18,51 @@ def apply_event(self, player):
-                        number = input("how many would you like to use? ")
-                        if number.isdigit() and int(number) > 0 and (int(number) <= player.inventory.items["weapon"]):
+                        number = input("Kuinka monta haluaisit käyttää? ")
+                        if number.isdigit() and int(number) > 0 and (int(number) <= player.inventory.items["ase"]):
                             number = int(number)
 
                             damage_after_weapon = 0
                             # damage cant be in plus
                             if value + (number * SETTINGS["weapon_damage"]) < 0:
                                 damage_after_weapon = value + (number * SETTINGS["weapon_damage"])
-                            setattr(player, 'health', getattr(player, 'health', 0) + damage_after_weapon)
+                            setattr(player, 'terveys', getattr(player, 'terveys', 0) + damage_after_weapon)
```

Ruff (0.8.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)